en/sim_gazebo_gz/README.md

```diff
@@ -192,13 +243,28 @@ Here are some examples of the different scenarios covered above.
    PX4_SYS_AUTOSTART=4001 PX4_GZ_MODEL_NAME=x500 ./build/px4_sitl_default/bin/px4
    ```
 
+4. **Start simulator in standalone mode + connect to Gazebo instance running default world**
+
+   ```sh
+   PX4_GZ_STANDALONE=1 PX4_SYS_AUTOSTART=4001 PX4_SIM_MODEL=gz_x500 ./build/px4_sitl_default/bin/px4
+   ```
+
+   In a separate terminal run:
+
+   ```sh
+   python /path/to/simulation-gazebo
+   ```
+
 ## Adding New Worlds and Models
 
-New worlds files can simply be copied into the PX4 Gazebo [world directory](https://github.com/PX4/PX4-Autopilot/tree/main/Tools/simulation/gz/worlds).
+SDF files, mesh files, textures and anything else to do with the functionality and appearance in Gazebo for worlds and models can be placed in the appropriate `/worlds` and `/models` directories in [PX4-gazebo-models](https://github.com/PX4/PX4-gazebo-models).
+
+Within PX4 follow the below steps to add models and worlds.
+
+### Adding a Model
 
 To add a new model:
 
-1. Add an **sdf** file in the PX4 Gazebo [model directory](https://github.com/PX4/PX4-Autopilot/tree/main/Tools/simulation/gz/models).
 1. Define an [airframe configuration file](../dev_airframes/adding_a_new_frame.md).
 1. Define the default parameters for Gazebo in the airframe configuration file (this example is from [x500 quadcopter](https://github.com/PX4/PX4-Autopilot/blob/main/ROMFS/px4fmu_common/init.d-posix/airframes/4001_gz_x500)):
```

Comment:
Below is pretty good, but probably not quite enough to understand how to create your own airframe file for Gazebo.

Specifically, there are other things in the file you link to such as `param set-default SIM_GZ_EC_FUNC1` and these aren't mentioned. 

It would be good to have an understanding of end to end how you create a frame for posix, answering questions like "can I have a single frame for both posix and nuttx?"